pkgs/by-name/go/gopeed/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, fetchurl
+, stdenv
+, autoPatchelfHook
+, dpkg
+, at-spi2-atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk3
+, harfbuzz
+, libayatana-appindicator
+, libayatana-indicator
+, libdbusmenu
+, pango
+}:
+stdenv.mkDerivation rec {
+  pname = "gopeed";
+  version = "1.5.7";
+
+  src = fetchurl {
+    url = "https://github.com/GopeedLab/gopeed/releases/download/v${version}/Gopeed-v${version}-linux-amd64.deb";
+    hash = "sha256-VIRxn+iHkJSzlX/RB4DsCHgB4+pGM3rJ2SE5ZAJwGwE=";
+  };
+
+  dontConfigure = true;
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ autoPatchelfHook dpkg ];
+
+  buildInputs = [
+    stdenv.cc.cc
+    at-spi2-atk
+    cairo
+    gdk-pixbuf
+    glib
+    gtk3
+    harfbuzz
+    libayatana-appindicator
+    libayatana-indicator
+    libdbusmenu
+    pango
+  ];
+
+  unpackPhase = ''
+    dpkg-deb -x ${src} ./
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mv usr $out
```

Comment:
Also, we don't normally just stick `usr` in `$out` like this

Reply:
Sorry but how to proceed  from there? Could you elaborate more on the  appropriate approach

Reply:
move the contents of `/usr` over rather than the folder itself